### ISSUE TITLE
'DbContextOptionsBuilder' does not contain a definition for 'UseSqlServer' and no extension method 'UseSqlServer' accepting a first argument of type 'DbContextOptionsBuilder' could be found

### ISSUE THREAD

**Issue by livarcocc** [dotnet/efcore#7970]
_From @Smilefounder on March 23, 2017 12:49_

## Steps to reproduce
Run the following command line:
`dotnet ef dbcontext scaffold "Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename=...\Database.mdf;Integrated Security=True;Connect Timeout=30"  Microsoft.EntityFrameworkCore.SqlServer --context DbContext --force --output-dir ../_tmp --json`

## Expected  behavior
POCO Generator and build without error.

## Actual behavior
Build error:
`1>_tmp\DbContext.cs(21,28,21,40): error CS1061: 'DbContextOptionsBuilder' does not contain a definition for 'UseSqlServer' and no extension method 'UseSqlServer' accepting a first argument of type 'DbContextOptionsBuilder' could be found (are you missing a using directive or an assembly reference?)`

## Environment data
`dotnet --info` output:

>dotnet --info
.NET Command Line Tools (1.0.0)

Product Information:
 Version:            1.0.0
 Commit SHA-1 hash:  e53429feb4

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.14393
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\1.0.0

_Copied from original issue: dotnet/cli#6126_

**Comment by ajcvickers** [comment]
@Smilefounder The output directory needs to be a project folder, not just a temporary file location. Also, naming your DbContext "DbContext" is likely to cause build issues--choose a different name.

**Comment by Smilefounder** [comment]
Thanks @ajcvickers for the answer, let me try again and keep you posted.
